### PR TITLE
temp_promos

### PR DESCRIPTION
**Pull request by ps2-controller**
API reference for MVP of temp_promos

### CODE REVIEW

File: responses/AccountTempPromos.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "description": "Array of all temporary promos for an account",
+  "content": {
+    "application/json": {
+      "schema": {
+        "$ref": "../schemas/AccountTempPromos.json"
+      }
+    }
```
Comment (dangdennis) [inline responses/AccountTempPromos.json:8]: can you stack multiple temp promos in the same period?

Comment (ps2-controller) [reply responses/AccountTempPromos.json:8]: In the Linear ticket, for now we've added:
```
risks: assume promos will not overlap (for now -- we can protect against this later)
```

Comment (ps2-controller) [reply responses/AccountTempPromos.json:8]: The system won't break if someone tries, but it'll just keep scheduling events over one another

File: schemas/AccountTempPromoItem.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        "late_fee": {
+          "type": "integer",
+          "description": "The late fee to be charged for late payments during the promotional period. ",
+          "example": 50000,
+          "default": 0
+        }
+      }
+    }
+  }
+}
```
Comment (goodgoodwish) [inline schemas/AccountTempPromoItem.json:40]: input would be sth like this:
```
{                                                    +
         "end_at": "2022-05-01T00:00:00-07:00",           +
         "promo_id": 1,                                   +
         "start_at": "2021-10-01T00:00:00-07:00",         +
         "account_id": 1,                                 +
         "updated_at": "2021-07-28T13:22:35.158388-07:00",+
         "config_name": "interest_rate",                  +
         "config_value": 0.12                             +
}
```

output of get() would be sth like this:
```
 [                                                        +
     {                                                    +
         "end_at": "2022-05-01T00:00:00-07:00",           +
         "promo_id": 1,                                   +
         "start_at": "2021-10-01T00:00:00-07:00",         +
         "account_id": 1,                                 +
         "updated_at": "2021-07-28T13:22:35.158388-07:00",+
         "config_name": "interest_rate",                  +
         "config_value": 0.12                             +
     },                                                   +
     {                                                    +
         "end_at": "2023-05-01T00:00:00-07:00",           +
         "promo_id": 2,                                   +
         "start_at": "2022-10-01T00:00:00-07:00",         +
         "account_id": 1,                                 +
         "updated_at": "2021-07-28T13:22:35.158548-07:00",+
         "config_name": "interest_rate",                  +
         "config_value": 0.08                             +
     },                                                   +
     {                                                    +
         "end_at": "2022-05-01T00:00:00-07:00",           +
         "promo_id": 3,                                   +
         "start_at": "2021-10-01T00:00:00-07:00",         +
         "account_id": 1,                                 +
         "updated_at": "2021-07-28T13:22:35.158554-07:00",+
         "config_name": "late_fee",                       +
         "config_value": 0                                +
     }                                                    +
 ]
```

Comment (goodgoodwish) [reply schemas/AccountTempPromoItem.json:40]: I thought again for the create() input.  Explicit config name as key works. such as:  ```{"late_fee_cents": 500}```

**Comment by dangdennis** [comment]
done!

**Review by dangdennis — APPROVED** [review]

**Pull request merged**